lib/util/http/github.ts

```diff
@@ -338,7 +338,7 @@ export class GithubHttp extends Http<GithubHttpOptions> {
               nextUrl.searchParams.set('page', String(pageNumber));
               return this.request<T>(
                 nextUrl,
-                { ...opts, paginate: false },
+                { ...opts, paginate: false, repoCache: false },
```

Comment:
If you mean `etagCache` option, it works in "bring your own backend" manner, i.e. you store latest `etag` + `data`, provide that to `http` layer, and if the server response was 304, then older data is being returned (or newer otherwise).

Reply:
Never mind, I haven't seen your previous PR